EzMap.Domain/Migrations/20231008092541_UpdateSchema_EzMap.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EzMap.Domain.Migrations
+{
+    /// <inheritdoc />
+    public partial class UpdateSchema_EzMap : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Poi_User_UserId",
+                table: "Poi");
+
+            migrationBuilder.AlterColumn<Guid>(
+                name: "UserId",
+                table: "Poi",
+                type: "uniqueidentifier",
+                nullable: true,
```

Comment:
Why is this nullable?
In that case need to rethink the model and identify the source of PoI. When will a PoI is in our system:

1. Do we import all PoIs from external provider (e.g. gg-map) into our database? This is kind of unrealistic though. But this allows UserId to be nullable.
2. Do we save a PoI only when a user save it? In that case PoI must have UserId (not nullable) as the user who saved/owns it.
3. Do we allow somehow a PoI can be imported to our db without a user interaction? If yes then UserId is nullable, but please elaborate which scenario.